happymonk-ai/Ckdir-frontend-
# ft homepage update

What does this PR do?
• Update homepage as per design in https://www.figma.com/proto/M1MJPoycrd0sXLBWdDuBvL/Happymonk-Website-V-3.0-Landing-Page?page-id=131%3A2572&node-id=181-3114&viewport=31%2C324%2C0.08&scaling=min-zoom&starting-point-node-id=131%3A4162

How should this be manually tested?
• Clone this repo
• Check out this branch ft-homepage-update
• Run yarn install to install all packages
• Run yarn dev to start the development environment, check the website live in browser (localhost:3000)

Any background context you want to provide?
• No additional background context is required for this PR.
